app/models/miq_action.rb

```diff
@@ -221,6 +222,11 @@ def action_audit(_action, rec, inputs)
                     :message      => "Policy #{msg}: policy: [#{inputs[:policy].description}], event: [#{inputs[:event].description}]")
   end
 
+  def action_run_ansible_playbook(action, rec, inputs)
+    service_template = ServiceTemplate.find(action.options[:service_template_id])
+    Api::ServiceTemplateWorkflow.create(service_template, {}).submit_request
```

Comment:
I was thinking the same thing.  This change would also support a recent request to enable service ordering from an automate methods. https://www.pivotaltracker.com/story/show/140055813